### PR TITLE
RBAC List users

### PR DESCRIPTION
**Pull request by soupette**
#### Description of what you did:

This PR lists the users. However it is still not connected to the API

### CODE REVIEW

File: packages/strapi-admin/admin/src/components/Users/List/utils/headers.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  },
+  {
+    name: 'username',
+    value: 'username',
+  },
+  {
+    // eslint-disable-next-line react/prop-types
+    cellAdapter: ({ isActive }) => {
+      return (
+        <>
```
Comment (HichamELBSI) [inline packages/strapi-admin/admin/src/components/Users/List/utils/headers.js:29]: Useless empty tags

File: packages/strapi-admin/admin/src/containers/Users/ListPage/index.js [hunk 2]
```diff
@@ -1,17 +1,149 @@
...
+          <FilterPicker onChange={handleChangeFilter} />
+          <Padded right size="10px" />
+          {filters.map((filter, i) => (
+            // eslint-disable-next-line react/no-array-index-key
+            <Filter key={i} {...filter} onClick={handleClickDeleteFilter} />
+          ))}
+        </Flex>
+      </BaselineAlignement>
       <ModalForm isOpen={isModalOpened} onToggle={handleToggle} />
+      <div style={{ height: 18 }} />
```
Comment (HichamELBSI) [inline packages/strapi-admin/admin/src/containers/Users/ListPage/index.js:138]:
```html
<BaselineAlignment top size="8px" />
<Padded top size="sm">
     <List ... />
</Padded>
```

File: packages/strapi-admin/admin/src/components/Users/Filter/index.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { Option } from '@buffetjs/core';
+
+const Filter = ({ filter, name, onClick, value }) => {
+  const { formatMessage } = useIntl();
+  const label = (
+    <>
+      <span>{name}</span>
+      <span style={{ fontWeight: 700 }}>
```
[the remark this answers is not in the document]
Comment (soupette) [reply packages/strapi-admin/admin/src/components/Users/Filter/index.js:11]: The bold fontWeight corresponds to 700

File: packages/strapi-admin/admin/src/components/Users/List/Wrapper.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+// TODO: this component is used to provide a class to the Table component from buffet in order to reset the style
```
[the remark this answers is not in the document]
Comment (soupette) [reply packages/strapi-admin/admin/src/components/Users/List/Wrapper.js:1]: We need to improve the Table in Buffet.js

File: packages/strapi-admin/admin/src/components/Users/FilterPicker/index.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+import PropTypes from 'prop-types';
+import { Picker } from '@buffetjs/core';
+import BaselineAlignment from '../../BaselineAlignement';
+
+import Button from './Button';
+import Card from './Card';
+
+const FilterPicker = ({ onChange }) => {
+  return (
+    <BaselineAlignment bottom size="6px">
```
[the remark this answers is not in the document]
Comment (soupette) [reply packages/strapi-admin/admin/src/components/Users/FilterPicker/index.js:11]: In fact here it is used to keep the same height as the `Option` component.
I have added the feedback to the codebase though.

**Pull request merged**